compilator/compilator_test.go

```diff
@@ -100,6 +114,10 @@ func TestCompilationBasic(t *testing.T) {
 	case <-time.After(1 * time.Second):
 		assert.Fail("Timed out waiting for overall completion")
 	}
+
+	contents, err := ioutil.ReadFile(metrics)
+	assert.Nil(err)
```

Comment:
Use `assert.NoError` when testing against errors (it gives a better message on failure).